### DISCUSSION TITLE
18.06无法通过web升级21.02

### DISCUSSION THREAD

**Discussion by sqnight-mm6** [immortalwrt/immortalwrt#693] [Q&A] [answered]
一台MT7621设备VR500，开始刷了ImmortalWrt 21.02.0，然后web升级到18.06-k5.4，现在无法通过web升回21.02，除了升级18.06，上传其他任何可用固件，都提示“不支持所上传的映像文件格式”，请问是什么问题？

**Comment by 1715173329** [comment, marked as answer]
supported_device 设定不一样，可以用命令行刷入
`sysupgrade -F -n <文件名>`
